routes/candidates_routes.js

```diff
@@ -13,6 +13,31 @@ c_router
         next()
     }) 
     
+    .get('/:candidateId', async function(req, res, next) {
+        const collection = database.db.collection("candidate")
+        const o_id  = req.params.candidateId
+        const item = await collection.findOne({candidateId : o_id })
+        console.log(item)
+        res.send(item)
+        next()
+    })
+
+    .post('/update', async function (req, res, next) {  
+        const collection = database.db.collection("candidate")
+        const fname = req.body.candidate_firstname
+        const cid = req.body.candidateId
+        console.log('fname is :' , fname)
+        console.log('cid is :',cid )
+        try {
+            collection.updateOne({ "candidate_firstname": fname }, { $set: { "candidateId": cid } });
+            res.send('updated')
+        }
+        catch (e) {
+            print(e)
```

Comment:
Where is this `print` coming from? Shouldn't this be `console.log`? Maybe `console.error` is better?
Also note on catch we don't return 500 and we don't call next, so the request will hang forever